### PR TITLE
doc/adding editorconfig and hl-column-fill documentation

### PR DESCRIPTION
**Pull request by jeetelongname**
Added docs for editorconfig and hl-column-fill

----

#

### CODE REVIEW

**Comment by rushsteve1** [comment]
This is related to #1166

File: modules/ui/fill-column/README.org [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+#+BEGIN_SRC emacs-lisp
+(setq-hook! 'major-mode-hook fill-column 80)
+#+END_SRC
+eg.
+#+BEGIN_SRC emacs-lisp
+(setq-hook! 'org-mode-hook fill-column 80)
+#+END_SRC
+
+After you need to simply restart ~hl-fill-column-mode~ if you already have
+those types buffers open
```
Comment (hlissner) [inline modules/ui/fill-column/README.org:65]: Add period at end of sentence.

File: modules/ui/fill-column/README.org [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+If you want to change the face use
+#+BEGIN_SRC emacs-lisp
+(after! hl-fill-column
+  (set-face-attribute 'hl-fill-column-face nil
+                      :background "YOUR-FONT-VALUE"
+                      :foreground "#YOUR-BACKGROUND-VALUE"
+                      :inverse-video t))
+#+END_SRC
+You can look at the documentation for ~set-face-attribute~ for the full list
+of customisation's
```
Comment (hlissner) [inline modules/ui/fill-column/README.org:76]: `s/customisation's/customizations`

Comment (hlissner) [reply modules/ui/fill-column/README.org:76]: And add period afterwards.

File: modules/ui/fill-column/README.org [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+#+END_SRC
+eg.
+#+BEGIN_SRC emacs-lisp
+(setq-hook! 'org-mode-hook fill-column 80)
+#+END_SRC
+
+After you need to simply restart ~hl-fill-column-mode~ if you already have
+those types buffers open
+** Configuring the look of the indicator
+If you want to change the face use
```
Comment (hlissner) [inline modules/ui/fill-column/README.org:67]: Add colon to end of the sentence

File: modules/ui/fill-column/README.org [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+#+BEGIN_SRC emacs-lisp
+(after! hl-fill-column
+  (set-face-attribute 'hl-fill-column-face nil
+                      :background "YOUR-FONT-VALUE"
+                      :foreground "#YOUR-BACKGROUND-VALUE"
+                      :inverse-video t))
```
Comment (hlissner) [inline modules/ui/fill-column/README.org:73]: Do not recommend `set-face-attribute` (or any `set-face-*` function) to users. Recommend `custom-set-faces!` instead, which is much less error prone (doesn't require the user to be mind load order of the `hl-fill-column` package and the theme. e.g.

```elisp
(custom-set-faces!
  '(hl-fill-column-face :background "red"
                        :foreground "blue"
                        :inverse-video t))
```

File: modules/ui/fill-column/README.org [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+#+END_SRC
+You can look at the documentation for ~set-face-attribute~ for the full list
```
Comment (hlissner) [inline modules/ui/fill-column/README.org:75]: Mention <kbd>SPC</kbd><kbd>h</kbd><kbd>f</kbd> `custom-set-faces\!` for looking up documentation and demos of this macro's usage.

File: modules/ui/fill-column/README.org [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+buffer. This is mainly to remind people to keep lines short for accessibility
+and or convenience. For more info read this [[https://www.emacswiki.org/emacs/EightyColumnRule][emacs wiki page]]
+* Configuration
+** Setting fill-column globally
+If you want to change the limit then you would need to set ~fill-column~ to your
+desired column you would do this using
+#+BEGIN_SRC emacs-lisp
+(setq-default fill-column 100)
+#+END_SRC
+For it to take affect you need to do..
```
Comment (hlissner) [inline modules/ui/fill-column/README.org:49]: Replace incomplete ellipses (`..`) at the end of the sentence with colon `:`)

File: modules/ui/fill-column/README.org [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
++ [[https://github.com/laishulu/hl-fill-column][hl-fill-column]]
+
+* Prerequisites
+This module has no prerequisites.
+
+* Features
+This module highlight's the characters that touch the 80th column of text in a
+buffer. This is mainly to remind people to keep lines short for accessibility
+and or convenience. For more info read this [[https://www.emacswiki.org/emacs/EightyColumnRule][emacs wiki page]]
+* Configuration
```
Comment (hlissner) [inline modules/ui/fill-column/README.org:42]: Please add a blank line before first-level headings.

**Review by hlissner — CHANGES_REQUESTED** [review]

File: modules/ui/fill-column/README.org [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+** Plugins
++ [[https://github.com/laishulu/hl-fill-column][hl-fill-column]]
+
+* Prerequisites
+This module has no prerequisites.
+
+* Features
+This module highlight's the characters that touch the 80th column of text in a
+buffer. This is mainly to remind people to keep lines short for accessibility
+and or convenience. For more info read this [[https://www.emacswiki.org/emacs/EightyColumnRule][emacs wiki page]]
```
Comment (hlissner) [inline modules/ui/fill-column/README.org:37]: Correct grammar:

> This module highlights the characters at the 80th column in the current buffer. This serves to remind users to keep lines short for accessibility and/or convenience. [[https://www.emacswiki.org/emacs/EightyColumnRule][Read more about this on the Emacs Wiki page]].

File: modules/ui/fill-column/README.org [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+* Configuration
+
+** Setting fill-column globally
+If you want to change the limit then you would need to set ~fill-column~ to your
+desired column you would do this using
```
Comment (hlissner) [inline modules/ui/fill-column/README.org:43]: Add colon `:` to end of the sentence.

File: modules/ui/fill-column/README.org [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+#+BEGIN_SRC emacs-lisp
+(setq-default fill-column 100)
+#+END_SRC
+For this to take effect in the current session cycle
```
Comment (hlissner) [inline modules/ui/fill-column/README.org:47]: Was something cut off here? Perhaps it should be 

> For this to take effect in the current session restart `hl-fill-column-mode`.

File: modules/ui/fill-column/README.org [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+** Setting fill-column on a mode by mode basis
+Setting on a mode by mode basis is quite simple
```
Comment (hlissner) [inline modules/ui/fill-column/README.org:50]: Add colon `:` to end of the sentence.

File: modules/ui/fill-column/README.org [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+If you want to change the face use:
+#+BEGIN_SRC emacs-lisp
+(after! hl-fill-column
+  (custom-set-faces! 'hl-fill-column-face nil
+                      :background "red"
+                      :foreground "blue"
+                      :inverse-video t))
+#+END_SRC
+You can look at the documentation for ~custom-set-faces!~ for the full list
+of customizations. This can be accessed using =SPC h f=
```
Comment (hlissner) [inline modules/ui/fill-column/README.org:73]: Add period to end of the sentence.

**Review by hlissner — CHANGES_REQUESTED** [review]

**Review by hlissner — APPROVED** [review]

**Pull request merged**